### ISSUE TITLE
Upgrading to kernel v3.16.1-7-ev3dev fails with error about rtl8192cu-modules

### ISSUE THREAD

**Issue by dlech** [ev3dev/ev3dev#211]
Yes, sorry about that. There are some dependency problems that don't quite work themselves out correctly with `apt-get update`. Here is the error that you will probably see.

```
Unpacking rtl8192cu-modules-common (4.0.2-9000.20131029-0ev3dev2+1~1) ...
dpkg: error processing archive /var/cache/apt/archives/rtl8192cu-modules-common_4.0.2-9000.20131029-0ev3dev2+1~1_all.deb (--unpack):
 trying to overwrite '/etc/modprobe.d/blacklist-native-rtl8192.conf', which is also in package rtl8192cu-modules-3.16.1-6-ev3dev 4.0.2-9000.20131029-0ev3dev1+1
```

Running `apt-get -f install` should fix the problem. For more info, see #192.

**Comment by dlech** [comment]
Since there has been an image released with kernel v3.16.1-7-ev3dev already installed, people shouldn't be running into this anymore.

**Comment by moriarty** [comment]
Thank you- I just picked up my ev3 for the first time in a while, late last night, quickly ran into this error, and decided to call it a night. 

Woke up this morning and this answer, which was here all along, was in my  inbox as a github alert.  
Everything is up to date now. :+1:
